<?xml version="1.0"?>
<launch>
  
  
  <include file="$(find robotican_demos)/launch/armadillo_find_objects_demo.launch"/>
  
  <node name="pan_tilt_object_tracking_node" pkg="robotican_common" type="pan_tilt_object_tracking_node" output="screen">
    <param name="~max_rate" value="0.3"/>
    <param name="~object_frame" value="object_frame"/>
    <param name="~depth_camera_frame" value="kinect2_depth_optical_frame"/>
    <param name="~kp_pan" value="4.0"/>
    <param name="~kp_tilt" value="3.0"/>
    <param name="~kd_pan" value="0.01"/>
    <param name="~kd_tilt" value="0.01"/>
    <param name="~loop_hz" value="30"/>
    <param name="~target_tol" value="0.05"/>  <!-- ~3 deg -->
    <param name="~no_object_timeout" value="5.0"/>
    
  </node>
  
  
  
  
</launch>

